src/defines.asm

```diff
@@ -1,38 +1,39 @@
 !ram_load_preset = $7FFB00
-
-!ram_gametime_room = $7FFB02
-!ram_last_gametime_room = $7FFB04
-!ram_realtime_room = $7FFB06
-!ram_last_realtime_room = $7FFB08
-!ram_last_room_lag = $7FFB0A
-!ram_last_door_lag_frames = $7FFB0C
-!ram_transition_counter = $7FFB0E
-!ram_transition_flag = $7FFB10
-!ram_transition_flag_2 = $7FFB12
-
-!ram_seg_rt_frames = $7FFB14
-!ram_seg_rt_seconds = $7FFB16
-!ram_seg_rt_minutes = $7FFB18
-!ram_reset_segment_later = $7FFB1A
-
-!ram_ih_controller = $7FFB1C
-!ram_slowdown_controller_1 = $7FFB1E
-!ram_slowdown_controller_2 = $7FFB20
-!ram_slowdown_frames = $7FFB22
+!ram_custom_preset = $7FFB02
+
+!ram_gametime_room = $7FFB04
+!ram_last_gametime_room = $7FFB06
+!ram_realtime_room = $7FFB08
+!ram_last_realtime_room = $7FFB0A
+!ram_last_room_lag = $7FFB0C
+!ram_last_door_lag_frames = $7FFB0E
+!ram_transition_counter = $7FFB10
+!ram_transition_flag = $7FFB12
+!ram_transition_flag_2 = $7FFB14
+
+!ram_seg_rt_frames = $7FFB16
+!ram_seg_rt_seconds = $7FFB18
+!ram_seg_rt_minutes = $7FFB1A
```

Comment:
Lol just love moving stuff around eh?

So I know these RAM values are undocumented, but I think someone is actually pulling these three values (frames/seconds/minutes) out of the practice rom.  IIRC it's trying to track your best segment time... or maybe it's the room times up above.

Anyway, if we don't have to change these values, I don't think we should.  It's good we did it for that big reorg a little while ago where defines were very disorganized, but not sure we should keep moving them around.